models/public/resnet-101/resnet-101.md

```diff
@@ -0,0 +1,64 @@
+# resnet-101
+
+## Use Case and High-Level Description
+
+[ResNet-101](https://arxiv.org/pdf/1512.03385.pdf)
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 14.441        |
+| MParams           | 44.496        |
+| Source framework  | Caffe\*       |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `data`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`. 
```

Comment:
as I know, resnet-50, resnet-101, resnet-152 have mean values:
`
Mean values [104.0, 117.0, 123.0]
`
but I do not see them in list_topologies.yml

Reply:
Yes, me too. I thought its a feature, but it may be bug in `list_topologies.yml`. Are you sure about mean values?

Reply:
In our local repo we don't have means for these topologies, so they are not exist in list_topologies

Reply:
So, there are no means for these models? Can it be resolved?

Reply:
@allnes wrote configs for OpenCV launcher in AC and tested models from model downloader. Mean values are present in his configs.

Reply:
I've created https://github.com/opencv/open_model_zoo/pull/208